CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.9.3 (2020-02-11)
+
+- Fix textarea readonly padding
```

Comment:
as pastanaga is an open source lib used by other than us, I added section formatting in last version. Can you follow the structure and add a section `Bug fixes` in your change log please? (I know it's a bit weird when there is only one fix, but I think it's better to be consistent)